@dgryski now a different build error, this time on Windows:

```
build/tinygo build -o test.elf -gc=leaking -scheduler=none examples/serial
error: Building Windows binaries is currently not supported. Try specifying a different target
mingw32-make: *** [Makefile:426: smoketest] Error 1
```

https://dev.azure.com/tinygo/tinygo/_build/results?buildId=3770&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774&t=632617cc-49d1-5f82-37d2-600ea5832b67&l=411

--- a/Makefile
+++ b/Makefile
@@ -451,6 +451,9 @@ endif
 	@$(MD5SUM) test.nro
 	$(TINYGO) build -size short -o test.hex -target=pca10040 -opt=0     ./testdata/stdlib.go
 	@$(MD5SUM) test.hex
+ifneq ($(OS),Windows_NT)
+	$(TINYGO) build -o test.elf -gc=leaking -scheduler=none examples/serial
+endif
 
 
 wasmtest:

